packages/myst-to-typst/src/container.ts

```diff
@@ -104,6 +104,11 @@ export const containerHandler: Handler = (node, state) => {
     return;
   }
 
+  if (node.enumerator?.endsWith('.1')) {
```

Comment:
This could be improved. It places strict assumptions on the `enumerator` in `numbering`. e.g. It works great for:
```
numbering:
  enumerator: 3.%s
```

But doesn't work at all for:
```
numbering:
  enumerator: (3-%s)
```

Reply:
Possibly we need better translation between myst and typst enumerators.